packages/uhk-web/src/styles-dark.scss

```diff
@@ -0,0 +1,12 @@
+@import './styles/variables';
+@import './styles/themes/dark';
+@import './styles/common';
+@import '~angular-notifier/styles';
+@import './styles/tooltip';
+@import './styles/uhk-icons/uhk-icon';
+@import './styles/side-menu';
+@import './styles/ngx-select-ex';
+
+@import './styles/bootstrap';
+
+@import './styles/global';
```

Comment:
The selected content same in the dark and light mode. Maybe worth move them in separated file and import this file in the light and dark mode css